BetterPasswordSettingsForm.inc.php

```diff
@@ -37,16 +37,26 @@ function __construct($plugin, $contextId) {
 
 		parent::__construct($plugin->getTemplateResource('settingsForm.tpl'), $this->_contextId);
 
-		$lockFields = array();
+		$lockFields = [];
+		$invalidationFields = [];
 		foreach (array_keys($this->_plugin->settingsKeys) as $key) {
 			if (strpos($key, 'betterPasswordLock') === 0) {
 				$lockFields[] = $key;
+			} elseif (strpos($key, 'betterPasswordInvalidation') === 0) {
+				$invalidationFields[] = $key;
 			}
 		}
-
+ __('plugins.generic.betterPassword.manager.settings.betterPasswordLockRequired');
```

Comment:
Forgotten line (I was checking the locale).
I've already updated locally the typos that I saw, but I'll wait your comments before pushing updates.